### PR TITLE
Added DragonAttenuation to fidelity tests

### PR DESCRIPTION
**Pull request by asemanh**
Added the images for reference

babylon-golden
filament-golden
gltf-sample-viewer-golden
model-viewer-golden

### CODE REVIEW

**Comment by elalish** [comment]
This is a good example of one we could reduce the height of. More than half the image is just background.

File: packages/render-fidelity-tools/test/config.json [hunk 1]
```diff
@@ -374,6 +374,16 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-DragonAttenuation",
```
Comment (elalish) [inline packages/render-fidelity-tools/test/config.json:378]: Also, this model is pretty (in my opinion), so let's put it toward the top.

**Review by elalish — CHANGES_REQUESTED** [review]

File: packages/render-fidelity-tools/test/config.json (hunk 1 above)
Comment (asemanh) [reply packages/render-fidelity-tools/test/config.json:378]: Agreeeed! I love it myself!

**Comment by asemanh** [comment]
babylon-golden
filament-golden
gltf-sample-viewer-golden
model-viewer-golden

File: packages/render-fidelity-tools/test/config.json [hunk 2]
```diff
@@ -25,6 +25,20 @@
     }
   ],
   "scenarios": [
+    {
+      "name": "khronos-DragonAttenuation",
```
Comment (elalish) [inline packages/render-fidelity-tools/test/config.json:29]: Okay, maybe not the *first* model :) This is a bit complicated (in terms of glTF extensions) and not all the renderers are even showing it properly yet. So, we don't need to showcase it right at the top, but at least in the upper section of pretty models.

File: packages/render-fidelity-tools/test/config.json [hunk 3]
```diff
@@ -25,6 +25,20 @@
...
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/DragonAttenuation/glTF/DragonAttenuation.gltf",
+      "dimensions": {
+        "height": 120,
+        "width": 130
```
Comment (elalish) [inline packages/render-fidelity-tools/test/config.json:33]: Sorry, this wasn't what I meant. Let's keep the resolution around what we usually use; I just don't want lots of blank pixels. For this one, I'm realizing it'll probably make more sense to zoom in a bit as we don't need to see the whole drape. Let's just focus on the dragon and frame that.

**Review by elalish — CHANGES_REQUESTED** [review]

File: packages/render-fidelity-tools/test/config.json (hunk 2 above)
Comment (asemanh) [reply packages/render-fidelity-tools/test/config.json:29]: sorry I missed this

**Review by elalish — APPROVED** [review]

**Pull request merged**